assets/css/extra.css

```diff
@@ -23,6 +23,13 @@ font-size of other elements in the theme.
 .rst-versions .rst-other-versions {
   color: #a9a9a9 !important;
 }
+
+/* adding a larger type option for lead text
```

Comment:
@garaynakca looks like you missed adding the closing comment `*/` to this line. I think the CSS won't work till that is added.